tests/test_sourmash.py

```diff
@@ -72,6 +72,74 @@ def test_sourmash_info_verbose():
     assert "loaded from path" in err
 
 
+def test_load_pathlist_from_file_does_not_exist():
+    from sourmash.sourmash_args import load_pathlist_from_file
+    with pytest.raises(ValueError) as e:
+        load_pathlist_from_file("")
+        assert "cannot open file ''" in e.message
```

Comment:
note, you should de-indent here, because the `assert` statement is never reached - this is because `load_pathlist_from_file` raises a ValueError, which ends the block and is caught by the `with` statement.

Reply:
Which lines should I de-indent?

Reply:
I'll ask a question instead ;).

try putting an `assert 0` after the `load_pathlist_from_file` call - does the test fail, as you'd expect it would?

also see:

- [asserting with the `assert` statement](https://docs.pytest.org/en/stable/assert.html#asserting-with-the-assert-statement)
- [`with pytest.raises` documentation](https://docs.pytest.org/en/stable/assert.html#assertions-about-expected-exceptions)
- [context managers](https://www.python.org/dev/peps/pep-0343/#id36

[python exceptions and try/except blocks](https://www.w3schools.com/python/python_try_except.asp) may also be useful.

And note that you can put `print` statements in tests and the output will be displayed only when the test fails, so they're useful for debugging tests.